description: 'Home calibration: arm fully extended upward'
joint_offsets:
  joint_1: 2.806306785430508
  joint_2: -2.2725929666744307
  joint_3: 3.7229414871835513
  joint_4: 0.4468971384647742
  joint_5: -0.3000417816323831
  joint_6: 0.8546873771183848
